kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Worker.java

```diff
@@ -54,26 +54,19 @@ public class Worker extends Channel {
 
   private final Catalog catalog;
   private final IcebergSinkConfig config;
-  private final Map<String, IcebergWriter> writers;
+  private final Map<String, IcebergWriter> writers = new HashMap<>();
   private final SinkTaskContext context;
   private final String controlGroupId;
-  private Set<String> dynamicTableList;
+  private final Map<String, Boolean> tableExistsMap = new HashMap<>();
 
   public Worker(Catalog catalog, IcebergSinkConfig config, SinkTaskContext context) {
-    super("worker", config);
+    // pass transient consumer group ID to which we never commit offsets
+    super("worker", "cg-control-" + UUID.randomUUID(), config);
```

Comment:
I see that we use the `cg-control-<uuid>` format in a few places, so it might be nice to have a util/prefix, but it also doesn't look like the format actually is used to reference anything (so maybe not necessary?).

Reply:
yes, I'll pull that out into a constant

Reply:
This is done